src/main/java/com/linkedin/kmf/apps/SingleClusterMonitor.java

```diff
@@ -61,6 +64,7 @@ public SingleClusterMonitor(Map<String, Object> props, String name) throws Excep
   @Override
   public void start() {
     _topicManagementService.start();
+    _commitAvailabilityService.start();
     CompletableFuture<Void> completableFuture = _topicManagementService.topicManagementResult();
     completableFuture.thenRun(() -> {
```

Comment:
I know this line is not introduced as part of this PR. But I am just wondering why making the start of the produceService and consumerService async? Is the goal here to make the method `start()` return faster? I think so services their `start()` methods called synchronously whereas others don't. It feels a little strange.

Another problem with having async start is that what if `_produceService.start();` or `_consumeService.start();` throws exception(s)? No code is going to handle it and the whole `SingleClusterMonitor` seems to have started. But actually not (inconsistent state).

Reply:
Please reference the previous Pull Requests for justifications to asynchronous calls. They have to do with the fact that  `_topicManagementResult` has to `complete()` within `MultiClusterTopicManagementService` class before the `produceService` and `consumeService` can start otherwise the ordering of these calls get disoriented causing failures.;